packages/server/postgres/migrations/1685721573097_addPrePostMortemTemplates.ts

```diff
@@ -0,0 +1,1110 @@
+    type: 'premortem',
+    prompts: [
+      {
+        question: '📣 Internal communication',
+        description: 'What communication issues might arise within the team?'
+      },
+      {
+        question: '💬 External communication',
+        description: 'What communication issues might arise with stakeholders or clients?'
+      },
+      {
+        question: '📩 Information flow',
+        description:
+          'What issues might hinder the flow of information between team members or stakeholders?'
+      },
+      {
+        question: '🤯 Misunderstandings',
+        description: 'What potential misunderstandings or misconceptions could cause problems?'
+      }
+    ]
+  },
+  {
+    name: 'Scrum sprint pre-mortem',
+    type: 'premortem',
+    prompts: [
+      {
+        question: '🎯 Product backlog',
+        description: 'What challenges could arise in maintaining a healthy and prioritized backlog?'
+      },
+      {
+        question: '📊 Sprint planning',
+        description: 'What obstacles might we face in effectively planning and executing sprints?'
+      },
+      {
+        question: '🔄 Retrospectives',
+        description:
+          'What challenges could impact the effectiveness of our retrospectives and continuous improvement efforts?'
+      },
+      {
+        question: '🚀 Sprint review',
+        description:
+          'What issues could affect our ability to showcase our progress to stakeholders?'
+      }
+    ]
+  },
+  {
+    name: 'Scrum roles pre-mortem',
+    type: 'premortem',
+    prompts: [
+      {
+        question: '🏃 Scrum Master',
+        description: 'What challenges might the Scrum Master face in facilitating the project?'
+      },
+      {
+        question: '👨‍💻 Developers',
+        description: 'What challenges might the developers face in executing the project?'
+      },
+      {
+        question: '📚 Product Owner',
+        description:
+          'What challenges might the Product Owner face in managing the product backlog and aligning with stakeholders?'
+      }
+    ]
+  }
+]
+
+const createdAt = new Date()
+
+const makeId = (name: string, type: 'template' | 'prompt') => {
+  const cleanedName = name
+    .replace(/[^0-9a-zA-Z ]/g, '') // remove emojis, apostrophes, and dashes
+    .split(' ')
+    .map(
+      (name, idx) =>
+        (idx === 0 ? name.charAt(0).toLowerCase() : name.charAt(0).toUpperCase()) + name.slice(1)
+    )
+    .join('')
+    .trim()
+  return `${cleanedName}${type === 'template' ? 'Template' : 'Prompt'}`
+}
+
+const makeTemplate = (template: Template): ReflectTemplate => ({
+  createdAt,
+  id: makeId(template.name, 'template'),
+  isActive: true,
+  name: template.name,
+  orgId: 'aGhostOrg',
+  scope: 'PUBLIC',
+  teamId: 'aGhostTeam',
+  type: 'retrospective',
+  updatedAt: createdAt,
+  isStarter: false,
+  isFree: true,
+  illustrationUrl:
+    template.type === 'postmortem' ? 'postMortemTemplate.png' : 'preMortemTemplate.png',
```

Comment:
-1 To get a full URL can you inline this function into the migration & run those names through it? That way it'll work for the cloud & PPMIs that use their own CDN

https://github.com/ParabolInc/parabol/blob/21a4b77d1f29e80be2bc51dd6a8bb5c920accade/packages/server/graphql/mutations/helpers/getTemplateIllustrationUrl.ts#L2-L14